game/templates/game/board.html

```
@@ -239,9 +559,13 @@
                     <span class="time" id="blackTime">10:00</span>
                 </div>
             </div>
+
+
+            <button id="pauseBtn" class="btn btn-gold">Pause</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)